data/json/items/magazine/9mm.json

```diff
@@ -324,5 +324,101 @@
     "reliability": 7,
     "reload_time": 140,
     "flags": [ "MAG_COMPACT" ]
+  },
+  {
+    "id": "bhp9mag_13rd",
+    "type": "MAGAZINE",
+    "name": "Hi-Power magazine 9x19mm",
+    "description": "A 13 round steel box magazine for the Browning Hi-Power 9x19mm.",
+    "weight": 100,
+    "volume": "110 ml",
+    "price": 2400,
+    "material": "steel",
+    "symbol": "#",
+    "color": "light_gray",
+    "ammo_type": "9mm",
+    "capacity": 13,
+    "reliability": 9,
+    "flags": [ "MAG_COMPACT" ]
+  },
+  {
+    "id": "bhp9mag_15rd",
+    "type": "MAGAZINE",
+    "name": "Hi-Power magazine 9x19mm",
+    "description": "A 15 round steel box magazine for the Browning Hi-Power 9x19mm.",
+    "weight": 100,
+    "volume": "120 ml",
+    "price": 2700,
+    "material": "steel",
+    "symbol": "#",
+    "color": "light_gray",
+    "ammo_type": "9mm",
+    "capacity": 15,
+    "reliability": 9,
+    "flags": [ "MAG_COMPACT" ]
+  },
+  {
+    "id": "p38mag",
+    "type": "MAGAZINE",
+    "name": "P38 magazine",
+    "description": "An 8 round steel box magazine for the Walther P38.",
+    "weight": 70,
+    "volume": "80 ml",
+    "price": 1500,
+    "material": "steel",
+    "symbol": "#",
+    "color": "light_gray",
+    "ammo_type": "9mm",
+    "capacity": 8,
+    "reliability": 9,
+    "flags": [ "MAG_COMPACT" ]
+  },
+  {
+    "id": "ppq9mag_10rd",
+    "type": "MAGAZINE",
+    "name": "PPQ magazine 9x19mm",
+    "description": "A 10 round steel box magazine for the Walther PPQ 9mm.",
+    "weight": 80,
+    "volume": 1,
+    "price": 2200,
+    "material": "steel",
+    "symbol": "#",
+    "color": "light_gray",
+    "ammo_type": "9mm",
+    "capacity": 10,
+    "reliability": 9,
+    "flags": [ "MAG_COMPACT" ]
+  },
+  {
+    "id": "ppq9mag_15rd",
+    "type": "MAGAZINE",
+    "name": "PPQ magazine 9x19mm",
+    "description": "A 15 round steel box magazine for the Walther PPQ 9mm.",
+    "weight": 80,
+    "volume": 1,
+    "price": 2800,
+    "material": "steel",
+    "symbol": "#",
+    "color": "light_gray",
+    "ammo_type": "9mm",
+    "capacity": 15,
+    "reliability": 9,
+    "flags": [ "MAG_COMPACT" ]
+  },
+  {
+    "id": "ppq9mag_17rd",
+    "type": "MAGAZINE",
+    "name": "PPQ magazine 9x19mm",
+    "description": "A 17 round steel box magazine for the Walther PPQ 9mm.",
+    "weight": 80,
+    "volume": 1,
```

Comment:
```suggestion
    "volume": "250 ml",
```

Reply:
@anothersimulacrum Sorry to ping, but I just want to make sure, is there a move to change volume from an int to string - should I be watching for this everywhere?

Reply:
Don't worry about pinging me, it's no big deal. Yes, there is a move to change volume from an int to a string, and please do watch for it.
Doing it this way makes it much more clear just reading the JSON and for new contributors what exactly it means.

Reply:
I will add the correct volumes when I'm able (or feel like, which may be soon after this, or in about 12 hours) to do so.

Reply:
If you're going to make all their volumes based on their real world volumes, please do so. But if it's just converting from a number to a string, there's no real need to do so, because it would be easier to do so with a script. (The existing adjust_values.py script only needs minor adjustment to do this).